### ISSUE TITLE
[Feature Request] Let brightness be increased.

### ISSUE THREAD

**Issue by D2lod** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#427]
I would like a way to increase brightness, when a game doesn't let you do it. I have been looking for any way to make Assetto Corsa brighter in my Vive Pro. The only way to do it is with it's post-processing filter which hurts performance a lot, but looks much more real because it is the right brightness for a sunny day. It is not quite bright enough on the Vive Pro, and that hurts immersion.

The game's post-processing filter settings let you tweak exposure time, color saturation, and contrast, but I don't need it to be post-processing, when a monitor can do it without performance impact.

I would love a brightness/contrast/saturation menu in the video page, which can make the headset display brighter, even if it sacrifices quality somehow. 

I have tried tweaking the PP filter and haven't found a way to reduce the performance hit, then looked for other ways to change the brightness in Asseto Corsa, but could not find a workaround. I tried to the brightness fix which uses extended mode, but the change doesn't stay when you switch the mode back. You can only use Nvidia control panel to change brightness when in extended mode.

It is a negative to me that the Vive Pro has a screen that is not as bright as the original Vive. And it really does have an impact in outdoor sunny environments, like a race track at noon or in The Talos Principle. The environment doesn't feel as real without brightness. 

Here is a link to a reddit post showing a measured difference between the Pro and original Vive (Thanks): https://www.reddit.com/r/Vive/comments/8c4luc/vive_pro_vs_vive_screen_brightness_and_color/

**Comment by Kung-vr** [comment]
On the video tab under color adjustment you should be able to manually type in values greater than 100% into the textboxes there. If you don't want to end up with a tinted color use the same value in all three, for example Red: 150% Green: 150% Blue: 150%.

**Comment by D2lod** [comment]
My goodness, dear sir, you've done it.

That works!! I even thought about doing that with colors, but thought typing wouldn't work. I did try typing into the standard brightness field.

This does just what I wanted. I just tested it some more with numbers up to 165, and yep, it makes the world look brighter and doesn't hurt contrast in that bright scene.

Thank you for this awesome program. A saturation bar would be awesome, too, but I don't know how doable that is. It is just mind-blowing to have a brightness setting on the headset after all this time.
